src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { TodoItem } from '../TodoItem';
+import { Todo } from '../../types/Todo';
+import { useTodos } from '../../TodosContext';
+import { FilterStatus } from '../../types/FilterStatus';
+
+export const TodoList: React.FC = () => {
+  const { todos, setTodos, filterStatus } = useTodos();
+
+  const filteredTodos = todos.filter(todo => {
+    switch (filterStatus) {
+      case FilterStatus.Active:
+        return !todo.completed;
+      case FilterStatus.Completed:
+        return todo.completed;
+      default:
+        return true;
+    }
+  });
```

Comment:
Pass to the `TodoList` component already filtered todos, so it just accept todos and render them without any logic